ceph/rbd/workflows/cleanup.py

```diff
@@ -0,0 +1,123 @@
+import json
+from time import sleep
+
+from ceph.rbd.utils import exec_cmd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def pool_cleanup(client, pools, **kw):
+    """_summary_
+
+    Args:
+        rbd (_type_): _description_
+        pools (_type_): _description_
+    """
+    if kw.get("dir_name"):
+        exec_cmd(node=client, cmd="rm -rf {}".format(kw.get("dir_name")))
+
+    ceph_version = kw.get("ceph_version")
+    # mon_allow_pool_delete must be True for removing pool
+    if ceph_version and ceph_version >= 5:
```

Comment:
Agree, but I would like to retain it till we get some confirmation on whether we will retain existing nautilus tests as is, or we will need to adopt them to SDK or remove them completely, so that we can remove the existing RBD and RBD Mirror class.